src/bin/driver.rs

```diff
@@ -83,12 +83,12 @@ pub struct Settings {
     /// See [StreamTarget#miniconf]
     stream_target: StreamTarget,
 
-    /// Thermostat interlock settings.
+    /// Thermostat alarm settings.
```

Comment:
I would not mention thermostat so prominently here and elsewhere. The feature is not limited to thermostat.
Do mention it as an example.